JSDemos/Demos/TabPanel/Overview/description.md

```diff
@@ -1,5 +1,13 @@
-<<<<<<< HEAD
-The TabPanel is&nbsp;a&nbsp;UI component consisting of&nbsp;the Tabs and the MultiView UI components. It&nbsp;automatically synchronizes the selected tab with the currently displayed view and vice versa. In&nbsp;this demo, the appearance of&nbsp;views and tabs is&nbsp;defined by&nbsp;custom templates.
-=======
-The TabPanel is&nbsp;a&nbsp;UI component consisting of&nbsp;the Tabs and the MultiView components. It&nbsp;automatically synchronizes the selected tab with the currently displayed view and vice versa. In&nbsp;this demo, the appearance of&nbsp;views and tabs is&nbsp;defined by&nbsp;custom templates.
->>>>>>> e74b8529 (Cut off "UI" from "UI component" is certain cases (#612))
+The [TabPanel](/Documentation/ApiReference/UI_Components/dxTabPanel/) UI component consists of the [Tabs](/Documentation/ApiReference/UI_Components/dxTabs/) and [MultiView](/Documentation/ApiReference/UI_Components/dxMultiView/) components. TabPanel automatically synchronizes the selected tab with the currently displayed view and vice versa. This demo shows the following TabPanel configuration::      
+
+### Bind the TabPanel to Data       
+
+The [dataSource](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/#dataSource) property binds the TabPanel to data. To specify a custom template for all the TabPanel views, use the [itemTemplate](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/#itemTemplate). To do the same for the tabs, use [itemTitleTemplate](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/#itemTitleTemplate).
+
+### Switch Between the Tabs     
+
+You can configure navigation between tabs. Specify the [swipeEnabled](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/#swipeEnabled) to define whether to switch between views with the swipe gesture. You can also [loop](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/#loop) views and use the [animationEnabled](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/#animationEnabled) property to specify whether to animate the change of the selected view.
+
+To change the selected tab programmatically, use the [selectedIndex](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/#swipeEnabled) property that is the index of the currently selected tab in the [dataSource](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/#dataSource) or [items[]](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/items/) array. The [onSelectionChanged](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/#onSelectionChanged) allows you to specify a function that is executed when the selected tab changes. 
```

Comment:
> To change the selected tab programmatically, use the [selectedIndex](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/#swipeEnabled) property that is the index of the currently selected tab in the [dataSource](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/#dataSource) or [items[]](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/items/) array.

I split the sentence in two, it was too long. Also, I removed links to the dataSource and items properties because they should be already linked in the section on data binding.

```suggestion
To switch between tabs programmatically, use the [selectedIndex](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/#selectedIndex) property. It accepts the index of a tab in the **dataSource** or **items[]** array. The [onSelectionChanged](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/#onSelectionChanged) function allows you perform desired actions when the selected tab changes.
```